data/abilities/credential-access/7049e3ec-b822-4fdf-a4ac-18190f9b66d1.yml

```diff
@@ -14,14 +14,14 @@
           Invoke-Mimikatz -DumpCreds
         parsers:
           plugins.stockpile.app.parsers.katz:
-          - source: domain.user.name
+          - source: user.name
             edge: has_password
-            target: domain.user.password
-          - source: domain.user.name
+            target: user.password
+          - source: user.name
             edge: has_hash
-            target: domain.user.ntlm
-          - source: domain.user.name
+            target: user.ntlm_hash
```

Comment:
have you verified that this doesn't break the katz parser? I believe the katz parser may use the last bit of the fact name to determine what data to retrieve

Reply:
My testing shows it does not break the parser. However, after running some comparison tests, it appears a portion of the output is being stomped, namely `user.name` is being overwritten and the hash output is being replaced by the entire unencrypted password. 

This leads to a larger question if only using `major.minor` is viable, or if other parsers have to be altered.

Reply:
the katz parser only looks at the last tag, so `major.minor` should still work. The issue though is that the katz parser seems to expect the the last tag to be either `password`, `ntlm`, or `sha1`. So the parser would have to be updated